lmdeploy/pytorch/engine/engine.py

```diff
@@ -761,24 +766,23 @@ def __get_num_ignore_eos(seqs: SeqList):
                 self._running = running
                 self._inputs = inputs
 
-                with torch.cuda.stream(self.stream):
-                    await self._async_step_background(
-                        inputs=inputs,
-                        swap_in_map=schedule_output.swap_in_map,
-                        swap_out_map=schedule_output.swap_out_map,
-                        history_ids=history_ids,
-                        sampling_inputs=sampling_inputs,
-                        num_appendable_ids=num_appendable_ids,
-                        num_ignore_eos=num_ignore_eos,
-                        output_que=out_que,
-                    )
+                await self._async_step_background(
```

Comment:
"with torch.cuda.stream(self.stream):" is removed. Will it bring side effect?

Reply:
has been move out to:
https://github.com/InternLM/lmdeploy/blob/41f179ee4812d310132a107db011ec4a13d46670/lmdeploy/pytorch/engine/engine.py#L850
So context just need to be added once.